sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/PlanParserSuite.scala

```diff
@@ -1084,6 +1087,18 @@ class PlanParserSuite extends AnalysisTest {
       table("testcat", "db", "tab").select(star()).hint("BROADCAST", $"tab"))
   }
 
+  test("option hint") {
```

Comment:
let's add a JIRA prefix 

```suggestion
  test("SPARK-36680: option hint") {
```